src/main/java/appeng/integration/abstraction/IInvTweaks.java

```diff
@@ -21,9 +21,19 @@
 
 import net.minecraft.item.ItemStack;
 
+import appeng.integration.IIntegrationModule;
 
-public interface IInvTweaks
+
+public interface IInvTweaks extends IIntegrationModule
 {
 
-	int compareItems( ItemStack i, ItemStack j );
+	default int compareItems( ItemStack i, ItemStack j )
+	{
+		throw new UnsupportedOperationException();
+	}
+
+	class Stub extends IIntegrationModule.Stub implements IInvTweaks
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Rename this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2176)